inc/hubspot-api.php

```diff
@@ -552,4 +549,44 @@ private function createDeal($deal)
 
         return $id;
     }
+
+    public function hsFileUpload($files)
+    {
+        if($this->folder!=null) {
+        $hsFilesPath = array();
+            foreach ($files as $filesIndex => $fileValue) 
+            {
+                if (!$fileValue==false) {
+                    $post_url = "https://api.hubapi.com/files/v3/files/";
+                    $ch       = curl_init(); 
+                    if (!$ch==false) {

```

Comment:
Formatting is wrong should be:
`if (! $ch == false)`

but you can just write this as `$ch`
as without the ! it would be `$ch == true`
which can be written as `if ($ch)`